experiments/ViterBrain/scripts/napari_gui.py

```diff
@@ -1,45 +1,58 @@
 from pathlib import Path
+from matplotlib.pyplot import colorbar
 from networkx.algorithms.operators.unary import reverse
 import numpy as np
 import napari
 import pickle
 import networkx as nx
+from scipy.fft import next_fast_len
+from sklearn.manifold import trustworthiness
 from tqdm import tqdm
 import os
 from napari_animation import AnimationWidget
 from brainlit.utils.Neuron_trace import NeuronTrace
 import zarr
+from magicgui import widgets
 
 scale = [0.3, 0.3, 1]
 num = 0
 
 root_dir = Path(os.path.abspath(""))
 data_dir = os.path.join(root_dir, "data", "example")
-path = os.path.join(data_dir, "image_viterbrain.pickle")
+path = os.path.join(data_dir, "3-1-soma_viterbrain.pickle")
```

Comment:
I think it is important to also provide any of the data that these scripts need (e.g. 3-1-soma_viterbrain.pickle). Can you please collect all the files that someone would need to run these scripts, and zip it and put it in, say, experiments/ViterBrain/data? Make sure to give it a good file name and add comments and modify the ViterBrain [README](https://github.com/neurodata/brainlit/tree/develop/experiments/ViterBrain) accordingly.